src/applications/mhv-medical-records/components/CareSummaries/ProgressNoteDetails.jsx

```diff
@@ -115,27 +115,47 @@ ${record.note}`;
       />
       <DownloadingRecordsInfo allowTxtDownloads={allowTxtDownloads} />
 
-      <div className="test-details-container max-80">
+      <div className="test-details-container max-80" data-dd-privacy="mask">
```

Comment:
You don't need to add the mask around the entire section if you've got it around the individual elements.

Reply:
fixed.